tests/test_sr.py

```diff
@@ -2336,6 +2336,12 @@ def setUp(self):
     def test_sop_class_uid(self):
         assert self._report.SOPClassUID == '1.2.840.10008.5.1.4.1.1.88.33'
 
+    def test_evidence(self):
+        ref_evd_items = self._report.CurrentRequestedProcedureEvidenceSequence
+        assert len(ref_evd_items) == 1
+        with pytest.raises(AttributeError):
+            assert self.PertinentOtherEvidenceSequence
+
```

Comment:
A test for evidence from different studies wouldn't be a bad idea?

Reply:
Addressed with dbe4c7707a002b3c9e7698d2e5d509cff33a331f